governance/polls/Increase wstETH Supply Cap to USD 1.2M - February 19, 2024.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Increase wstETH Supply Cap to USD 1.2M - February 19, 2024
+summary: Signal your support or opposition to increasing the wstETH supply cap to USD 1.2M.
+discussion_link: https://forum.makerdao.com/t/feb-14-2024-proposed-changes-to-sparklend-for-upcoming-spell/23684
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2024-02-19T16:00:00
+end_date: 2024-02-22T16:00:00
+---
+# Poll: Increase wstETH Supply Cap to 1.2M - February 19, 2024
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of ecosystem actor Phoenix Labs. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, February 19 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following changes:
+
+- Increase [wstETH](https://app.spark.fi/reserve-overview/?underlyingAsset=0x7f39c581f595b53c5cb19bd0b3f8da6c935e2ca0&marketName=proto_spark_v3) [Supply Cap](https://docs.sparkprotocol.io/developers/sparklend/features/supply-borrow-caps#supply-caps) by 400,000 wstETH from 800,000 to **USD 1.2M** wstETH.
```

Comment:
It's preferable to use "million" instead of "M" for clarity

```suggestion
- Increase [wstETH](https://app.spark.fi/reserve-overview/?underlyingAsset=0x7f39c581f595b53c5cb19bd0b3f8da6c935e2ca0&marketName=proto_spark_v3) [Supply Cap](https://docs.sparkprotocol.io/developers/sparklend/features/supply-borrow-caps#supply-caps) by 400,000 wstETH from 800,000 to **USD 1.2 million** wstETH.
```